spec/features/user_sees_activities_page_spec.rb

```diff
@@ -25,7 +25,7 @@
     expect(page).to have_content("Register")
   end
 
-  it 'has a cart link that works' do
+  it 'has a cart_trips link that works' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.